### PR TITLE
UKTS-34 - Add airfield as an organisational unit

### PR DESCRIPTION
**Pull request by AxonC**
- Adds position as an organisational unit in preparation for training groups etc.

### CODE REVIEW

**Comment by AxonC** [comment]
Note: I did this rather quickly so may have missed smaller things.

File: tests/Feature/Position/PositionTest.php [hunk 1]
```diff
@@ -93,20 +103,22 @@ public function testItReturnsPositionByCallsign()
     /** @test */
     public function testValidPositionCanBeCreated()
     {
-        $this->graphQL('
+        $this->withoutExceptionHandling();
```
Comment (CalumTowers) [inline tests/Feature/Position/PositionTest.php]: Can this be removed?

**Review by CalumTowers — CHANGES_REQUESTED** [review]

File: tests/Feature/Position/PositionTest.php (hunk 1 above)
Comment (AxonC) [reply tests/Feature/Position/PositionTest.php]: of course! done

**Review by CalumTowers — DISMISSED** [review]

File: app/Modules/Position/Position.php [hunk 2]
```diff
@@ -29,13 +31,18 @@ class Position extends Model
         self::TYPE_TERMINAL => ['TMA', 'Terminal'],
         self::TYPE_FSS => ['FSS', 'Flight Service Station'],
     ];
-    protected $fillable = ['callsign', 'name', 'frequency', 'type'];
+    protected $fillable = ['callsign', 'name', 'frequency', 'type', 'airfield'];
```
Comment (atoff) [inline app/Modules/Position/Position.php]: best practice to have this property called `airfield_icao`, such that the relationship can be `airfield()`?

**Review by atoff — CHANGES_REQUESTED** [review]

File: app/Modules/Position/Position.php (hunk 2 above)
Comment (AxonC) [reply app/Modules/Position/Position.php]: yeah, I was in two minds about this, but you've convinced me

Comment (AxonC) [reply app/Modules/Position/Position.php]: all sorted

**Review by atoff — APPROVED** [review]

**Pull request merged**